e2e/davinci-suites/src/basic.test.ts

```diff
@@ -85,3 +85,30 @@ test('ensure query params passed to start are sent off in authorize call', async
   const accessToken = await page.locator('#accessTokenValue').innerText();
   expect(accessToken).toBeTruthy();
 });
+
+test('Enter a bad username/password, then enter a good username/password', async ({ page }) => {
+  await page.goto('http://localhost:5829/');
+  await page.getByRole('textbox', { name: 'Username' }).fill('baduser');
+  await page.getByRole('textbox', { name: 'Username' }).press('Tab');
```

Comment:
Why are we pressing "Tab" here?

Reply:
I recorded the test using playwright test recorder, so I must have pressed tab.